scripts/test-dssspell-forge.sh

```diff
@@ -18,7 +18,7 @@ done
 DSS_EXEC_LIB=$(< DssExecLib.address)
 echo "Using DssExecLib at: $DSS_EXEC_LIB"
 export FOUNDRY_LIBRARIES="lib/dss-exec-lib/src/DssExecLib.sol:DssExecLib:$DSS_EXEC_LIB"
-export FOUNDRY_OPTIMIZER=true
+export FOUNDRY_OPTIMIZER=false
```

Comment:
Confirming as reviewer that this is expected behavior as it reverts the changes from the previous spell, which required the optimizer in order to deploy / cast correctly.

Change is OK